serial_views.py

```diff
@@ -1,16 +1,24 @@
 # serialize data for all models
 import datetime
 from django.http import HttpResponse
+from django.views import View
 import sqlalchemy as sa
 import sqlahelper as sah
 import geojson
-from geojson import Point, Feature, FeatureCollection, dumps
+from geojson import Point, MultiPolygon, Feature, FeatureCollection, dumps
+from geoalchemy2.shape import from_shape
+from shapely.geometry import Point, shape
```

Comment:
Be careful! Here you import twice `Point`, I would stick to specify `geojson.Point`, or import it under another alias

Reply:
thats true I will try ro fix this, point is not a good name anyway - there are columns named point too.